Cargo.toml

```diff
@@ -1,38 +1,88 @@
+# Copyright (C) 2022 Nicolas Stalder AND  Nitrokey GmbH
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
 [package]
 name = "piv-authenticator"
-version = "0.0.0-unreleased"
-authors = ["Nicolas Stalder <n@stalder.io>"]
-edition = "2018"
+version = "0.3.2"
+authors = ["Nicolas Stalder <n@stalder.io>", "Nitrokey GmbH"]
+edition = "2021"
 license = "Apache-2.0 OR MIT"
-repository = "https://github.com/solokeys/piv-authenticator"
+repository = "https://github.com/nitrokey/piv-authenticator"
 documentation = "https://docs.rs/piv-authenticator"
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[[example]]
+name = "vpicc"
+required-features = ["vpicc"]
+
+
+[[example]]
+name = "usbip"
+required-features = ["apdu-dispatch", "virt"]
 
 [dependencies]
 apdu-dispatch = { version = "0.1", optional = true }
-delog = "0.1.0"
+delog = { version = "0.1.5", optional = true }
 flexiber = { version = "0.1", features = ["derive", "heapless"] }
 heapless = "0.7"
 hex-literal = "0.3"
-interchange = "0.2.0"
+interchange = "0.2.2"
 iso7816 = "0.1"
-serde = { version = "1", default-features = false }
-trussed = { git = "https://github.com/trussed-dev/trussed" }
-# trussed = { path = "../../../trussed" }
-untrusted = "0.7"
+serde = { version = "1", default-features = false, features = ["derive"] }
+trussed = { version = "0.1", features = ["serde-extensions"] }
+trussed-auth = { version = "0.2" }
+untrusted = "0.9"
+vpicc = { version = "0.1.0", optional = true }
+log = "0.4"
+heapless-bytes = "0.3.0"
+subtle = { version = "2", default-features = false }
+trussed-rsa-alloc = { version = "0.1.0", features = ["raw"] }
+trussed-staging = { version = "0.1.0", features = ["chunked", "encrypted-chunked"]}
 
 [dev-dependencies]
-littlefs2 = "0.3.1"
+littlefs2 = "0.3.2"
 rand_core = { version = "0.6", features = ["getrandom"] }
+trussed = { version = "0.1.0", features = ["virt"] }
+env_logger = "0.9"
+serde = { version = "1", features = ["derive"] }
+serde_cbor = { version = "0.11", features = ["std"] }
+hex  = "0.4"
+test-log = "0.2.11"
+ron  = "0.8"
+des = "0.8"
+aes = "0.8.2"
+stoppable_thread = "0.2.1"
+expectrl = "0.6.0"
+
+# Examples
+trussed-usbip = { git = "https://github.com/trussed-dev/pc-usbip-runner", default-features = false, features = ["ccid"], rev = "f3a680ca4c9a1411838ae0774f1713f79d4c2979"}
+usbd-ccid = { version = "0.2.0", features = ["highspeed-usb"]}
+rand = "0.8.5"
+asn1 = "0.15.2"
 
 [features]
 default = []
 strict-pin = []
+std = []
+vpicc = ["std", "dep:vpicc", "virt"]
+virt = ["std", "trussed/virt"]
+pivy-tests = []
+opensc-tests = []
 
 log-all = []
 log-none = []
 log-info = []
 log-debug = []
 log-warn = []
 log-error = []
+
+[patch.crates-io]
+trussed = { git = "https://github.com/trussed-dev/trussed" , rev = "55ea391367fce4bf5093ff2d3c79041d7aef0485" }
```

Comment:
These will have to be merged/moved upstream first

Reply:
I moved them.